smoke_screens/premium_home_screen.rb

```diff
@@ -0,0 +1,7 @@
+module SmokeScreens
+  class PremiumHomeScreen < BasicScreen
+    def path
+      '/premium/products'.freeze
```

Comment:
@jeyboy - the premium home is /premium

We also need to add premium products page /premium/products